### PR TITLE
Docs: Updated color for title & refined type sizes.

### PR DESCRIPTION
**Pull request by yuinchien**

### CODE REVIEW

**Comment by looeee** [comment]
[Live](https://raw.githack.com/mrdoob/three.js/type-hierarchy/docs/#manual/en/introduction/Creating-a-scene).

This is better, but I think it still needs a bit of work, especially on large screens.

* the line width of articles is too wide. Max ~75 chars should be the aim. Currently I'm counting 123 chars. 
* line spacing on menu items is too high. 
* a bit more differentiation between font choices in code/title/text would be good. 
* Roboto mono is used for the code, but it's not obviously mono. I think if the the difference from the main text was more obvious that would be less of a problem

I actually am using roboto mono myself, and people have made the same complaint to me. I guess it takes looking at another site using it to see why 😅 

Getting text readable and flowing well on all screen sizes is a pain. I've switched to using a technique called Fluid Typography on on Discoverthreejs.com, and it's made my life a lot easier.

For a quick intro read this [CSS-Tricks article](https://css-tricks.com/snippets/css/fluid-typography/). 
For more in depth analysis of the technique including best practices for scaling headings, text width, line height etc. this [Smashing Magazine article](https://www.smashingmagazine.com/2016/05/fluid-typography/) is worth reading. 

In the end this is all the CSS used for font-sizing on my entire site:

``` css
body {

  font-size: calc(16px + (18 - 16) * ((100vw - 400px) / (1200 - 400)));

}

pre {
  font-size: 0.9em;
}
```

File: docs/page.css [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 	--font-size: 14px;
 	--line-height: 24px;
-	--font-size-sanserif: 1.128rem;
-	--line-height-sanserif: 1.74rem;
+	--font-size-sanserif: 1.14285714286rem;
```
Comment (mrdoob) [inline docs/page.css]: Haha, how are you getting these numbers? 😁

**Comment by mrdoob** [comment]
@looeee 20 years later and CSS is still a mystery to me... 😁

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
I have yet to try on different screens, but in my current setup (phone/macbook) after this change it looks great on both.

@looeee's point about fonts being similar for code than text is interesting. I can see that a slight change of visual helps the brain to change contexts (basically reading different languages). 

Maybe instead of changing the font it would help just making the font-size a little bit smaller for the code snippets?

**Comment by looeee** [comment]
> @looeee 20 years later and CSS is still a mystery to me... 😁

Yeah, being a CSS expert is an entire career in itself...  😓

**Pull request merged**